samples/wifi/shell/src/wifi_cred_shell.c

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <zephyr/shell/shell.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/init.h>
+
+#include <zephyr/net/net_if.h>
+#include <zephyr/net/wifi_mgmt.h>
+#include <zephyr/net/net_event.h>
+
+#include <net/wifi_credentials.h>
+#include <net/wifi_mgmt_ext.h>
+
+#define MACSTR "%02hhx:%02hhx:%02hhx:%02hhx:%02hhx:%02hhx"
+
+static void print_network_info_helper(void *cb_arg, const char *ssid, size_t ssid_len)
```

Comment:
```suggestion
static void print_network_info(void *cb_arg, const char *ssid, size_t ssid_len)
```
This is subjective and of little consequence, of course, but the `_helper` does not seem necessary to me, and it's nice when functions sound like imperative statements. The command handlers are consistently prefixed `cmd_`, so there is no chance of confusion.

Reply:
done